Tests/MaterialLib/TestRelativePermeabilityModel.cpp

```diff
@@ -0,0 +1,163 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   TestRelativePermeabilityModel.cpp
+ *
+ * Created on November 2, 2016, 3:09 PM
+ */
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <vector>
+
+#include "BaseLib/ConfigTree.h"
+
+#include "TestTools.h"
+
+#include "MaterialLib/PorousMedium/UnsaturatedProperty/RelativePermeability/RelativePermeability.h"
+#include "MaterialLib/PorousMedium/UnsaturatedProperty/RelativePermeability/CreateRelativePermeabilityModel.h"
+
+using namespace MaterialLib;
+using namespace MaterialLib::PorousMedium;
+
+std::unique_ptr<RelativePermeability> createRelativePermeabilityModel(
+    const char xml[])
+{
+    auto const ptree = readXml(xml);
+    BaseLib::ConfigTree conf(ptree, "", BaseLib::ConfigTree::onerror,
+                             BaseLib::ConfigTree::onwarning);
+    auto const& sub_config = conf.getConfigSubtree("relative_permeability");
+    //! \ogs_file_attr{prj__material_property__porous_medium__porous_medium__id}
+    auto const id = sub_config.getConfigAttributeOptional<int>("id");
```

Comment:
id is reserved for material IDs of elements. We can not input the model parameters as Parameter data because that we may use different models for different material zones in a modelling.